sdk/react/src/context.ts

```diff
@@ -2,10 +2,10 @@ import { createContext } from 'react'
 import type { DVCClient, DVCVariable } from '@devcycle/devcycle-js-sdk'
 
 interface DVCContext {
-  client?: DVCClient
+  client: DVCClient
 }
 
-const context = createContext<DVCContext>({ client: undefined })
+const context = createContext<DVCContext>({ client: undefined! })
```

Comment:
you should avoid using `!` here as well. What you're basically doing here is telling typescript that the context will always have a client field of type "DVCClient", then basically promising the compiler here that `undefined` is actually a DVCClient.

Either you should keep the type of the context here the same or find a way to create it without passing undefined as the client